@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/mpd/fn_mpdRFCRDraw.sqf

```diff
@@ -11,9 +11,15 @@ if (_heli getVariable "fza_ah64_agmode" == 1) then {
 	_heli setobjecttexture [SEL_MPD_PR_MPD_HAD_APOS, ""];
 	_heli setobjecttexture [SEL_MPD_PR_MPD_HAD_APOS2, ""];
 } else {
-	_heli setobjecttexture [SEL_PR_MPD_BACK, "\fza_ah64_US\tex\mpd\grnd.paa"];
-	_heli setobjecttexture [SEL_MPD_PR_FCRAA_WIPER, ""];
-	_heli setobjecttexture [SEL_MPD_PR_FCRAG_WIPER, "fza_ah64_us\tex\mpd\grnd.paa"];
+	if (_heli animationphase "fcr_enable" == 1) then {
```

Comment:
What if by some way the agmode was set to 1?

I would recommend doing the same thing, but setting it using an exitWith at the top, for example at around line 3 do

```sqf
if (_heli animationphase "fcr_enable" == 0) exitWith {
    _heli setobjecttexture [SEL_PR_MPD_BACK, "\fza_ah64_US\tex\mpd\grnd.paa"];  //change this with FCR not installed backround
    _heli setobjecttexture [SEL_MPD_PR_FCRAA_WIPER, ""];
    _heli setobjecttexture [SEL_MPD_PR_FCRAG_WIPER, ""];
};
```